bazel/envoy_test.bzl

```
@@ -80,11 +80,17 @@ def envoy_cc_fuzz_test(
         name,
         corpus,
         dictionaries = [],
+        engflow_pool = None,
+        exec_properties = None,
```

```suggestion
        exec_properties = {},
```

and update below - same elsewhere